c/copy/copy_ubi_8.5.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : copy
+# Version       : v1.7.0
+# Source repo   : https://github.com/otiai10/copy
+# Tested on     : UBI: 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License 2.0
+# Maintainer's  : Balavva Mirji <Balavva.Mirji@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
```

Comment:
no need to set -e again as we are doing the same in the first line

Reply:
@seth-priya - updated the script.